pkg/fileutils/fileutils.go

```diff
@@ -98,7 +98,22 @@ func (pm *PatternMatcher) Matches(file string) (matches uint, err error) {
 		logrus.Debugf("Skipping excluded path: %s", file)
 	}
 
-	return matches, nil
+	return matches, excludes, nil
+}
+
+// IsMatch verifies the provided filepath against all patterns and returns true
+// if it overall matches. It returns an error on failure and is not safe to be
```

Comment:
"overall" is a bit loose. Can we describe exactly what it means? Some user might expect exclusions to yield an unmatch.

Reply:
Okay, I will extend the explanations.